cmd/key_rotate.go

```diff
@@ -86,6 +86,11 @@ func BuildCmdRotateKey(w io.Writer, handler RotateKeyHandler, rf *RootFlags) *co
 		"",
 		"A public key to rotate to. Should be generated by wallet's 'generate' command.",
 	)
+	cmd.Flags().StringVarP(&f.CurrentPubKey,
+		"current-pubkey", "c",
+		"",
+		"A public key to rotate from. Should be currently used public key.",
+	)
```

Comment:
No we have to specify the current pubkey, the `current-pubkey` is fine but the `pubkey` is not anymore. It should be name `new-pubkey`. Also, I would get rid of the short parameter `c` and `k`.